postgres/datadog_checks/postgres/statements.py

```diff
@@ -27,23 +27,21 @@
 
 STATEMENTS_QUERY = """
 SELECT {cols}
-  FROM {pg_stat_statements_view} as pg_stat_statements
+  FROM {pg_stat_hstatements_view} as pg_stat_statements
```

Comment:
was this an intentional change?

Reply:
oh wat. Not at all.

Reply:
I'm not sure how that happened (was my cat stepping on my keyboard?) but I fixed it.